app/models/lion.rb

```diff
@@ -23,6 +23,17 @@ def self.create_from_image_set(image_set, name)
     lion
   end
 
+  def self.search(search_params)
+    age_params = search_params.extract!(:age_range_start, :age_range_end)
+
+    unless age_params.empty?
+      dob_hash = { date_of_birth: (age_params[:age_range_end]..age_params[:age_range_start]) }
```

Comment:
@redyaffle  I think `start..end` is more intuitive, so the start would be the eldest possible age / furthest date in the past. Why not rename to `age_range_earliest_dob` and `age_range_latest_dob`